src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -1581,20 +1581,36 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
             // if the number of directories deep exceeds threshold
             if (str.Length - str.Replace(@"\", "").Length >= 5)
             {
```

Comment:
This is marginally too complicated for its own good :)

Given that we've seen this crash, I think we would be justified in extracting this into a helper method and writing a couple of trivial regression unit tests.